platformcomponents/desktop/attachment-file-preview-thumbnail.json

```diff
@@ -0,0 +1,12 @@
+{
+  "attachment-file-preview-thumbnail": {
+    "sent": {
+      "comment": "Applied to file attachments",
```

Comment:
Applied to file attachment preview thumbnail

Reply:
Pushed commit
`Changed comment for attachment-file-preview-thumbnail`
Changing conversation to Resolved.